tests/unit_tests/test_wave_02.py

```diff
@@ -32,9 +32,10 @@ def test_get_no_matching_items_by_category():
         inventory=[item_a, item_b, item_c]
     )
 
-    items = vendor.get_by_category("electronics")
+    items = vendor.remove("electronics")
 
-    raise Exception("Complete this test according to comments below.")
+    assert(items == False)
```

Comment:
`get_by_category` here should return an empty list:
```suggestion
    assert len(items) == 0
```